python/tests/catalog/test_glue.py

```diff
@@ -19,7 +19,6 @@
 
 import pytest
 from moto import mock_glue
-
```

Comment:
I like the line between the first and third party libraries. I think we can fix this by setting:
```
lines-between-types = 1
```

See doc: https://beta.ruff.rs/docs/settings/#isort-lines-between-types